api/geth_backend.go

```diff
@@ -153,6 +154,16 @@ func (b *GethStatusBackend) GetMultiaccountDB() *multiaccounts.Database {
 	return b.multiaccountsDB
 }
 
+func (b *GethStatusBackend) InitializeAccounts(rootDirectory string) error {
+	b.UpdateRootDataDir(rootDirectory)
+	manager := b.AccountManager()
+	keystoreDir := filepath.Join(rootDirectory, "keystore")
```

Comment:
I remember a recent PR from @Samyoul also had this "keystore" magic string. Could we extract it into a single place?

Reply:
We do have `common` packages that are used for these cases. At the moment my #3248 is still in testing on the client sides